### PR TITLE
Request several workflow/pipeline release perms for self

### PR DESCRIPTION
**Pull request by svanoort**
Since I am part of the team working on it regularly and the others likely to cut a release are already there. 

CC @jglick

### CODE REVIEW

File: permissions/plugin-workflow-basic-steps.yml [hunk 1]
```diff
@@ -5,4 +5,5 @@ paths:
 developers:
 - "jglick"
 - "kohsuke"
-- "abayer"
```
Comment (jglick) [inline permissions/plugin-workflow-basic-steps.yml]: Can we do the right thing here and put in the trailing newline? :-)

File: permissions/plugin-workflow-cps.yml [hunk 2]
```diff
@@ -5,4 +5,5 @@ paths:
 developers:
 - "jglick"
 - "kohsuke"
-- "abayer"
\ No newline at end of file
+- "abayer"
+- "svanoort"
```
Comment (jglick) [inline permissions/plugin-workflow-cps.yml]: Please run tests with `-Djenkins.version=2.7.3` before cutting releases here—high risk.

**Comment by jglick** [comment]
:+1:

**Comment by svanoort** [comment]
@jglick Done, noted, and thanks!

**Pull request merged**